Essentials/src/com/earth2me/essentials/commands/Commandessentials.java

```diff
@@ -307,6 +312,29 @@ private void run_uuidtest(final Server server, final CommandSource sender, final
         sender.sendMessage("Offline Mode UUID: " + offlineuuid.toString());
     }
 
+    private void run_version(final Server server, final CommandSource sender, final String commandLabel, final String[] args) throws Exception {
+        if (!sender.isPlayer() && ess.getUserMap().getUser(sender.getPlayer().getName()).isAuthorized("essentials.version")) return;
```

Comment:
This doesn't look right.

Reply:
Good catch; it isn't right. It's been a while but I think the `!` belongs in the second statement.

Reply:
Yes.

Reply:
I think I enabled push access for maintainers, so if anyone wants to fix it feel free, as I'm busy for the next few weeks.